x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/node/label_node/label_node_badges.tsx

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
```

Comment:
nit: you can save a lot of repeat code here by introducing a component of LabelBadge and re use it for all the different scenarios